kalite/updates/static/js/updates/bundle_modules/update_videos.js

```diff
@@ -415,10 +415,12 @@ function getSelectedStartedMetadata(data_type) {
 }
 
 function setNodeClass(node, className) {
-    if (node.extraClasses != className) {
-        node.extraClasses = className;
-        if (node.parent !== null) {
-            updateNodeClass(node.parent);
+    if (node !== null) {
```

Comment:
I don't think it does affect future readability of this code, as the code is badly constructed, poorly written, and non-idiomatic JS.

The best fix here is to :fire: down the code and start from scratch. Maybe we can with @benjaoming's 0.17 reworking.